src/unity/python/turicreate/toolkits/drawing_recognition/drawing_recognition.py

```diff
@@ -229,23 +219,35 @@ def export_coreml(self, filename, verbose=False):
                                 class_labels=self.classes,
                                 input_shape=[('bitmap', image_shape)],
                                 builder=None, verbose=verbose,
-                                preprocessor_args={'image_input_names':['bitmap']},
-                                is_drawing_recognition=True)
-
+                                preprocessor_args={'image_input_names':['bitmap']})
+
+        DESIRED_OUTPUT_NAME = "probabilities"
```

Comment:
Should use `self.target + 'Probabilities'`?

Reply:
Done